deploy/kubernetes/operator.yaml

```diff
@@ -3,6 +3,10 @@ kind: Deployment
 metadata:
   name: nsx-ncp-operator
   namespace: nsx-system-operator
+  labels:
+    tier: nsx-networking
+    component: nsx-ncp-operator
+    version: v1
 spec:
   replicas: 1
   selector:
```

Comment:
we shouldn't change
 matchLabels:
   name: nsx-ncp-operator

since matchLabels are the immutable filed, it can not be directly updated for running pods, so, we have to delete old pods to make it effective. but it will make rolling update failed.